dataframely/columns/_base.py

```diff
@@ -79,6 +95,42 @@ def validate_dtype(self, dtype: PolarsDataType) -> bool:
 
     # ---------------------------------- VALIDATION ---------------------------------- #
 
+    def _get_rule_names(self, rules: list[Callable[[pl.Expr], pl.Expr]]) -> list[str]:
```

Comment:
This is really only about _check_ rules. Could we name this accordingly, e.g.

```suggestion
    def _derive_check_rule_names(self, rules: list[Callable[[pl.Expr], pl.Expr]]) -> list[str]:
```

Reply:
And [nit], can we move this method below the `validation_rules` method (within the `VALIDATION` section)? I find it easier to read through code if the most important stuff comes first :smile: